### DISCUSSION TITLE
UI Filtering instantly blanks out results when composing, making iteration very difficult

### DISCUSSION THREAD

**Discussion by bamboowonder** [directus/directus#12858] [Feature Requests]
### Describe the Bug

While composing UI Filter criteria, the data is filtered instantly/prematurely and table left blank.

This UX pattern is frustrating, since in the process of composing the filter, especially a nested or multilayer,  it is necessary to see the actual data *before* filtering so know *what* to filter for.   

Iterating and refining the filter then becomes dependent on my memory of what data there was without a filter.

The text box Search being instant is fine as it is usually a single value that is known in advance with no iteration.

But Filtering benefits a lot from the ability to iterate and refine to hone in on what is needed.  

A workaround for this, and great improvement on it's own, would be a checkbox per line in the filter to temporarily disable a line of the filter, instead of having to delete to see results again.

Or simply the option to not trigger the filter until I hit a button would be helpful.

### To Reproduce

In the UI, begin a filter on a table with a lot of columns and rows.   Disengage your photographic memory if you have one.
Table instantly goes blank.
Erase filter to see results again, try to remember what you want to filter or write it down.
Then begin filtering again and hope you got it right..

### What version of Directus are you using?

9.9

### What version of Node.js are you using?

16.14.2

### What database are you using?

postgres 14

### What browser are you using?

chrome

### How are you deploying Directus?

CLI npm package

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
